test/OpenTelemetry.Instrumentation.W3cTraceContext.Tests/W3CTraceContextTests.cs

```diff
@@ -57,9 +64,89 @@ public void W3CTraceContextTestSuite(string value)
                 string lastLine = ParseLastLine(result);
                 this.output.WriteLine("result:" + result);
                 Assert.StartsWith("FAILED", lastLine);
+#if NETCOREAPP2_1
+                AssertOutputOfNetCoreApp21(result);
+#else
+                AssertOutputOfNetCoreApp31(result);
+#endif
+            }
+        }
+
+        private static void AssertOutputOfNetCoreApp21(string output)
+        {
+            string[] unsuccessfulTestcaseNames = ExtractUnsuccessfulTestcaseNames(output);
+
+            // TODO: fix and remove current failures except for expected failures. #1219
+            // See expected failed or error cases in https://github.com/open-telemetry/opentelemetry-dotnet/issues/404
+            string[] currentFailures = new string[]
```

Comment:
rename to expectedFailures?

Reply:
Changed it to `existingFailures` for less misunderstanding. The failures are either pending investigation or due to known issues in external dependencies. We don't "expect" them to fail but only calling them out in order to get them fixed later.